### PR TITLE
add reference link in mesheryctl

### PR DESCRIPTION
**Pull request by Althaf66**
**Notes for Reviewers**
@leecalcote @alphaX86 
This PR adds a reference link to the mesheryctl.

It helps to understand the commands in better way by visiting the documentation.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Yay!  A first-time PR, @Althaf66 🎈

**Review by alphaX86 — CHANGES_REQUESTED** [review]
Changes are good @Althaf66 

However, FYI...
https://docs.meshery.io/project/contributing/contributing-cli#contributing-to-mesheryctl-documentation

Follow the changes as suggested from this docs page :) For time-being, the current documentation script follows this format to generate docs pages.

**Review by Althaf66 — COMMENTED** [review]
reviewed

File: mesheryctl/internal/cli/root/app/app.go [hunk 1]
```diff
@@ -40,9 +40,9 @@ var AppCmd = &cobra.Command{
 	Short: "Cloud Native Apps Management",
 	Long:  `All apps operations: import, list, view, onboard and offboard`,
 	Example: `
-// Base command
-mesheryctl app [subcommand]
-	`,
+Base command:
+  mesheryctl app [subcommand]
+Find more information at: https://docs.meshery.io/reference/mesheryctl#command-reference`,
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/app/app.go:45]:
```suggestion
Base command:
  mesheryctl app [subcommand]
Find more information at: https://docs.meshery.io/reference/mesheryctl#command-reference`,
```

If this format is used, then the corresponding docs page won't format in proper way. FYI, https://docs.meshery.io/reference/mesheryctl/main 

Suppose if you remove `//` then it'll treat as code block and print in wrong format, similarly the link you've provided will result in code block as well

File: mesheryctl/internal/cli/root/filter/filter.go [hunk 2]
```diff
@@ -30,11 +30,12 @@ var (
...
-	Long:  ``,
+	Long:  `Cloud Native Filter Management`,
 	Example: `
-// Base command for WASM filters
-mesheryctl filter [subcommands]	
-	`,
+Base command for WASM filters:
+  mesheryctl filter [subcommands]
+	
+Find more information at: https://docs.meshery.io/reference/mesheryctl#command-reference`,
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/filter/filter.go:38]: Same

File: mesheryctl/internal/cli/root/pattern/pattern.go [hunk 3]
```diff
@@ -33,18 +33,19 @@ var PatternCmd = &cobra.Command{
 	Short: "Cloud Native Patterns Management",
 	Long:  `Manage service meshes using predefined patterns`,
 	Example: `
-// Apply pattern file
-mesheryctl pattern apply --file [path to pattern file | URL of the file]
+Apply pattern file:
+  mesheryctl pattern apply --file [path to pattern file | URL of the file]
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/pattern/pattern.go:37]: Same

**Review by alphaX86 — CHANGES_REQUESTED** [review]
Revert changes please, also refer 1st review comment

File: mesheryctl/internal/cli/root/app/app.go (hunk 1 above)
Comment (alphaX86) [reply mesheryctl/internal/cli/root/app/app.go:45]: I advise you to set the link elsewhere and NOT in the examples field for a while

Comment (leecalcote) [reply mesheryctl/internal/cli/root/app/app.go:45]: Is there an example to follow or CLI contributing docs that cover this, @alphaX86?

Comment (Althaf66) [reply mesheryctl/internal/cli/root/app/app.go:45]: Hi,@alphaX86 @leecalcote, I understand the issue. As per meshery docs, link can print only after the example field. But for cobra-cli, we cannot add reference link below the example field. There is no particular option for that.

**Comment by saurabh100ni** [comment]
Thanks for raising PR @Althaf66, If you want discuss this PR, Meshery Development Meet is the ideal place for it. Please add this in [Docs](https://github.com/meshery/meshery/pull/9716/files) as your item to discuss it in the Meet today at 8:30 pm IST.
Check out [Community Calendar ](https://layer5.io/community/calendar)for meet Details.

File: mesheryctl/internal/cli/root/app/app.go (hunk 1 above)
Comment (leecalcote) [reply mesheryctl/internal/cli/root/app/app.go:45]: Familiarizing with this - https://docs.meshery.io/project/contributing/contributing-cli#contributing-to-mesheryctl-documentation - @Althaf66 might shed some new light on the issue.

**Comment by leecalcote** [comment]
@Althaf66 and @alphaX86 will you be on today's meeting? Can you guys hash this out on the call?

**Comment by Althaf66** [comment]
sorry @leecalcote,university semester exam will be on tommorrow. I can't attend the meeting today.

**Comment by leecalcote** [comment]
> sorry @leecalcote,university semester exam will be on tommorrow. I can't attend the meeting today.

Thanks for sending word.

**Comment by l5io** [comment]
🚀 Preview for commit fb9bba0883e6dc18650d5602131f292a5b0e6def at: https://65957b0a33849a0e990121b1--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8853b0166743aed6516a0514ecb55d514209c8b4 at: https://65957fe20e956311d8fd2a5f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3f4506dbe35b34816f0deaeb27dc97da0235e23a at: https://6595877f1e54c11194990e73--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a35a407a1846914379d0106cf69c6582efe6bf6f at: https://65958d4d1e54c115f0991238--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 14f698af257c7dc65e5e5e2d869ad132dc032248 at: https://65958e8f1e54c11728990c38--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/app/app.go (hunk 1 above)
Comment (alphaX86) [reply mesheryctl/internal/cli/root/app/app.go]: Ok, I've modified the `docs.go` file accordingly to reflect this. `mesheryctl` will show the ref link in the CLI but will stay hidden in the synopsis of command documentation pages.

**Review by leecalcote — APPROVED** [review]
Thank you, @alphaX86  and @Althaf66

**Review by alphaX86 — APPROVED** [review]
Thanks @Althaf66 we're good to go

**Pull request merged**